sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -6,10 +6,27 @@
 
 ### Breaking Changes
 
+- Merged `MultiCategoryClassifyResult` and `SingleCategoryClassifyResult` into `LabelClassifyResult`.
```

Comment:
what are we all calling this type? In Python I went with `ClassifyDocumentResult` 
@deyaaeldeen @mssfang

Reply:
Good catch. This was before I realized they were merged. I'll leave them as-is for this PR, but will fix them when I do a separate PR for merging them. I didn't want this PR to get too big. 😆

Reply:
Same in Java. "ClassifyDocumentResult"